pkg/kube/controllers/quarkssecret/quarkssecret_reconciler.go

```diff
@@ -165,7 +164,7 @@ func (r *ReconcileQuarksSecret) createPasswordSecret(ctx context.Context, qsec *
 	request := credsgen.PasswordGenerationRequest{}
 	password := r.generator.GeneratePassword(qsec.GetName(), request)
 
-	secret := &corev1.Secret{
+	secret := &v1.Secret{
```

Comment:
nit: most of our code uses `corev1` for that import

Reply:
Done.